monitoring/grafana-guide.md

```diff
@@ -0,0 +1,69 @@
+# Integrating Grafana dashboard
+
+This doc assumes that you have setup the necessary Prometheus resources
+to monitor dragonfly pods. Please refer to [pod-monitor-guide](podMonitorGuide.md)
+for further information on PodMonitor setup.
+
```

Comment:
Can you add a line about the `grafana-dashboard.json` itself for folks who already have a grafana instance in their cluster here?

Reply:
Sure